contrib/scalafix/README.md

```diff
@@ -0,0 +1,54 @@
+# Pants plugin to support executing scalafix
+
+A plugin to add support for executing the [scalafix](https://scalacenter.github.io/scalafix/) tool.
+
+## Setup
+
+Unlike `scalafmt` (which is included in the default distribution of pants), `scalafix` requires a
+compiler plugin, and is thus distributed as a contrib module with extra setup steps.
+
+The [scalameta](http://scalameta.org/) compiler plugin extracts semantic information at compile
+time that is used in some of the most useful `scalafix` rewrites. To install `scalameta` in your
+build, you should define a `//:scalac-plugin-dep` target (at the root of your repository) that
+references it, and uses the full compiler version:
+
+```
+FULL_SCALA_REV=2.11.11
+jar_library(
+  name = 'scalac-plugin-dep',
+  jars = [jar(org='org.scalameta', name='scalahost_{}'.format(SCALA_REV), rev='1.8.0')],
+)
+```
+
+Then, you will additionally need to enable plugin, and pass the `-Yrangepos` flag for scalac
+in `pants.ini`:
+```
+[compile.zinc]
+args: [
+    # The `-S` prefix here indicates that zinc should pass this option to scalac rather than
+    # to javac (`-C` prefix).
+    '-S-Yrangepos',
+  ]
+
+scalac_plugins: [
+    'scalahost',
+  ]
+```
+
+## Usage
+
+Tasks for both the `fmt` and `lint` goals are provided, each with the same set of options.
```

Comment:
What do you think about calling these goals `fix` and `check` in accordance with how these actions are called in Scalafix? https://github.com/scalacenter/scalafix/blob/master/scalafix-core/shared/src/main/scala/scalafix/rule/Rule.scala.